schema/snapcraft.json

```diff
@@ -922,6 +915,19 @@
                     "required": [
                         "build-base"
                     ]
+                },
+                {
+                    "usage": "type: gadget (with a base)",
```

Comment:
This section looks redundant now with:
`"usage": "base: <base> and type: <app|gadget>",`

Reply:
True!

Reply:
done